packages/config/src/validation.ts

```diff
@@ -119,20 +119,88 @@ export const isValidBrowserList = (_key: string, browsers: any): ErrResult | tru
   return true
 }
 
+const isValidExperimentalRetryOptionsConfig = (options: any, strategy: 'detect-flake-but-always-fail' | 'detect-flake-and-pass-on-threshold'): boolean => {
+  if (options === null) return true
```

Comment:
```suggestion
  if (options == null) return true
```

Or `options === null || options === undefined`, so it has the same behavior as the flipped version before when it was `options != null`